digitalocean/resource_digitalocean_volume.go

```diff
@@ -53,6 +53,12 @@ func resourceDigitalOceanVolume() *schema.Resource {
 				Optional: true,
 				ForceNew: true, // Update-ability Coming Soon ™
 			},
+
+			"filesystem_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true, // Update-ability Coming Soon ™
```

Comment:
Let's remove this comment. The DigitalOcean API does not currently offer a way to re-format a volume, so "update-ability" isn't something we can promise.